velociraptor/autoplotter/objects.py

```diff
@@ -754,61 +756,51 @@ def get_quantity_from_catalogue_with_mask(
         # in versions of unyt less than 2.6.0
         name = x.name
 
+        # temporary masked array to apply global mask
+        # in concert with plot-specific masks
+        x_mask = self.global_mask
+        
```

Comment:
Is it possible to remove these lines and replace `x_mask` with `self.global_mask` in the two `logical_and()` below in the code?

Reply:
Yes, this makes sense, and avoids modifying `x_mask` later, will implement